framework/wazuh/core/mitre.py

```diff
@@ -110,28 +134,11 @@ def __init__(self, table: str = None, offset: int = 0, limit: Union[int, None] =
             The value of the output dictionary will be the value of the remaining key.
         """
 
-        if filters is None:
-            filters = {}
-        if min_select_fields is None:
-            if table == 'phase':
-                self.min_select_fields = {'tactic_id', 'tech_id'}
-                default_sort_field = 'tactic_id'
-            elif table == 'mitigate' or table == 'use':
-                # source_id refers to mitigation_id, group_id or software_id
-                # target_id always refers to technique_id
-                self.min_select_fields = {'source_id', 'target_id'}
-                default_sort_field = 'source_id'
-        else:
-            self.min_select_fields = min_select_fields
-        if fields is None:
-            if table == 'phase':
-                fields = {'tactic_id': 'tactic_id', 'tech_id': 'tech_id'}
-            elif table == 'mitigate':
-                fields = {'source_id': 'source_id', 'target_id': 'target_id'}
-            elif table == 'use':
-                fields = {'source_id': 'source_id', 'source_type': 'source_type',
-                          'target_id': 'target_id', 'target_type': 'target_type'}
-        self.dict_key = dict_key if dict_key else next(iter(self.min_select_fields))
+        self.min_select_fields = min_select_fields or self.TABLE_FIELDS[table]['min_select_fields']
+        self.dict_key = dict_key or next(iter(self.min_select_fields))
```

Comment:
Is the `next(iter())` really what you want there? Consider this case from above:

```python
'use': {'min_select_fields': {'source_id', 'target_id'}
```

It would return only one item.

Reply:
Changed in favor of the default_sort_field